flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/operators/transform/PostTransformOperatorTest.java

```diff
@@ -780,6 +795,76 @@ void testTimestampTransform() throws Exception {
         transformFunctionEventEventOperatorTestHarness.close();
     }
 
+    @Test
+    void testUnixTimestampTransform() throws Exception {
```

Comment:
Please make this test parameterized in multiple timezones.

See MySqlTimezoneITCase for an example.